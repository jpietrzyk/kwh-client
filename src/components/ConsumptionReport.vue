<template>
  <time-range></time-range>
</template>

<script>
  import ReportTimeRange from '@/components/ReportTimeRange'
  export default {
    components: {
      'time-range': ReportTimeRange
    }
  }
</script>
