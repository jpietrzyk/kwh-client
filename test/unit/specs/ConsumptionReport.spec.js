import Vue from 'vue'
import ConsumptionReport from '@/components/ConsumptionReport'

describe('ConsumptionReport.vue', () => {
  it('should render correct contents', () => {
    const Constructor = Vue.extend(ConsumptionReport)
    const vm = new Constructor().$mount()
  })
})
