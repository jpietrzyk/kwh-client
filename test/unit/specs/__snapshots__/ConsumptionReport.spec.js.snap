// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConsumptionReport.vue has the expected html structure 1`] = `<!---->`;
